tests/testsuite/manifest.go

```diff
@@ -75,7 +76,7 @@ func ReadManifestYamlFile(pathToManifest string) []unstructured.Unstructured {
 	var objects []unstructured.Unstructured
 	stream, err := os.Open(pathToManifest)
 	util.PanicOnError(err)
-
+	defer putil.CloseIOAndCheckErr(stream, &err)
```

Comment:
There are two options to use this helper function:
- Passing `nil` as the error means that an error of the `Close` is to reflected back, just logged. This will usually happen if the function (e.g `ReadManifestYamlFile`) does not return an error.
I think this is what fits here: `defer putil.CloseIOAndCheckErr(stream, nil)`
- Passing a named return value of the error.
Example:
```go
func foo() (retErr error) {
    f, err := os.Open(myfile)
    if err != nil { return err }
    
    defer putil.CloseIOAndCheckErr(f, &retErr)
}
```

But what we really need here is something like this:
```go
defer func() {
    e := stream.Close()
    util.PanicOnError(e)
}()
```

Because on any error, these helper functions are raising panics intentionally.

Reply:
Thanks for the review and detailed explanation. I had some misunderstandings about CloseIOAndCheckErr before.

Reply:
Any reason you prefer logging over panic-ing?

Reply:
1. Failure to close the file does not affect subsequent processes
2. In the entire project, the error handling of close connect and close file is mostly recorded as debug or not handled, and no panic handling method is found.

Reply:
I think the panic operation is suitable for scenarios where an error occurs and subsequent functions cannot be used.

Reply:
Handling the `Close` error is borderline anyway, so this  reasoning sounds reasonable to me. Thanks.